ci/common/test-pyclaragenomics.sh

```diff
@@ -32,6 +32,8 @@ rsync -aP rsync://hgdownload-euro.soe.ucsc.edu/genome/admin/exe/linux.x86_64/bed
 rsync -aP rsync://hgdownload.soe.ucsc.edu/genome/admin/exe/linux.x86_64/bigWigToBedGraph $LOCAL_BIN_DIR/ ||
 rsync -aP rsync://hgdownload-euro.soe.ucsc.edu/genome/admin/exe/linux.x86_64/bigWigToBedGraph $LOCAL_BIN_DIR/
 
+pyflakes .
```

Comment:
let's move this check to the `ci/checks/style.sh` script and update to `flake8` instead of `pyflakes`. example 

https://github.com/clara-genomics/ClaraGenomicsAnalysis/blob/dev-v0.5.0/ci/checks/style.sh#L44
https://github.com/clara-genomics/ClaraGenomicsAnalysis/blob/dev-v0.5.0/pyclaragenomics/style_check

here's the python formatting config file
https://github.com/clara-genomics/ClaraGenomicsAnalysis/blob/dev-v0.5.0/pyclaragenomics/setup.cfg